pages/sources.md

```diff
@@ -141,6 +141,23 @@ mapbox:
     url: https://a.tiles.mapbox.com/v4/mapbox.mapbox-streets-v6-dev/{z}/{x}/{y}.vector.pbf?access_token=...
 ```
 
+##### MBTiles
+
+[[ES-only](https://github.com/tangrams/tangram-es)] If the _URL_ of a data source has a file extension of `.mbtiles` then the specified file will be accessed as an MBTiles database and used as a tile set according to version 1.1 of the MBTiles specification: https://github.com/mapbox/mbtiles-spec
```

Comment:
Instead of version 1.1, probably we should mention `according to the proposed version 2.0 of the MBTiles spec`.

Reply:
Yeah that makes more sense, updated.